src/webapp/default_config.py

```diff
@@ -0,0 +1,13 @@
+GIT_SSH_KEY = None
+
+TOPOLOGY_DATA_DIR = "/tmp/topology/topology"
+TOPOLOGY_DATA_REPO = "https://github.com/opensciencegrid/topology"
+TOPOLOGY_DATA_BRANCH = "master"
+
+CONTACT_DATA_DIR = "/tmp/topology/contact"
+CONTACT_DATA_REPO = "git@bitbucket.org:opensciencegrid/contact.git"
+CONTACT_DATA_BRANCH = "master"
+
+CACHE_LIFETIME = 60 * 15
```

Comment:
Two thoughts:

1.  Can we use anywhere besides `/tmp`?  I hate named files in `/tmp`.
2.  I'm OK if we bump the cache lifetime to 30 minutes.... I'd prefer to focus on getting webhooks working smoothly.

Reply:
1. Both of these directories are supposed to get overridden in the config files on the hosts (the config files I put on the topology host write to `/var/cache/topology` and `/var/cache/topology-itb`), so there are just defaults if you don't have a config file, e.g. when testing on a local machine. I wanted something that a user could write to.

2. How about if we bump the cache lifetime once we have the webhooks implemented?